CONTRIBUTING.rst

```diff
@@ -3,24 +3,20 @@ Contributing guide
 
 Thanks for your interest in contributing to cpplint.
 
-Any kinds of contributions are welcome: Bug reports, Documentation, Patches.
+Any kinds of contributions are welcome: Bug reports, Documentation, Patches. If you need some ideas, you may check out some of the tasks below:
 
-However, cpplint is a bit special as a project because it aims to closely follow what Google does in the upstream repository.
-That means Google remains the source of all major requirements and functionality of cpplint whereas this fork adds extensions to cpplint run on more environments and in more companies.
-The difference between this cpplint and Google should remain so small that anyone can at a glance see there is no added change that could be regarded as a security vulnerability.
+* Reducing interference with other popular tools
+* Update cpplint for modern python versions
+  * Converting setup.py to a pyproject.toml
```

Comment:
We need to be careful here - this isnt difficult to do badly, and inviting people to do this will likely result in it being done badly.  We want to maintain Python backwards compatibility as much as possible because people using cpplint are likely to be using much older versions of Windows/Linux for their build environment because they have some third-party SDK that isnt available on newer OS versions.

Reply:
AFAIK pyproject.toml is compatible with 3.6.
How about
```suggestion
* Preserve backwards-compatibility without sacrificing readability
* Update cpplint for modern python versions
  * Converting setup.py to a pyproject.toml
```

Reply:
My point is this document is not the place to ask for someone to do "Converting setup.py to a pyproject.toml".
An issue should be created about this enhancement, and the needs to be done carefully according to that issue, probably by one of the maintainers.

Reply:
I'm not sure how to incorporate this now.